nvflare/private/fed/server/server_engine.py

```diff
@@ -570,17 +571,17 @@ def _get_aux_msg_target(self, name: str):
         else:
             return None
 
-    def send_aux_to_targets(self, targets, topic, request, timeout, fl_ctx, optional, secure):
+    def _to_aux_msg_targets(self, target_names: List[str]):
```

Comment:
Shall we rename aux_msg to stream_msg?

Reply:
No, this is a utility method that turns target names to target FQCNs. This code used to be part of the aux messaging, and I refactored it so it can be used for both aux messaging and aux-msg based streaming.